### PR TITLE
Minor fixes to matlab named-user license changes

### PR DESCRIPTION
**Pull request by gpacix**
Remove write permissions on mlm.opt;
fix iteritems --> items throughout;
fix compat-libstdc++-33 package name to work with CentOS 7 and 6

### CODE REVIEW

File: deploy/library/postgresql_exec.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def main():
         "port":"port",
         "db":"database"
     }
-    kw = dict( (params_map[k], v) for (k, v) in module.params.iteritems()
+    kw = dict( (params_map[k], v) for (k, v) in module.params.items()
```
Comment (tanderegg) [inline deploy/library/postgresql_exec.py:138]: Was this to support Python 3?  I believe it is usually actually run on Python 2.x in our environment, and in that environment iteritems is more efficient, but if we need to change it to support 3, that's fine too.

File: deploy/roles/odbc/templates/odbcinst.ini [hunk 2]
```diff
@@ -30,7 +30,7 @@ useNTLMv2               = True
 
 {% for driver in odbcinst_entries %}
 [{{ driver.drivername }}]
-{% for key, value in driver.attributes.iteritems() %}
+{% for key, value in driver.attributes.items() %}
```
Comment (tanderegg) [inline deploy/roles/odbc/templates/odbcinst.ini:33]: See comment above

File: deploy/library/postgresql_exec.py (hunk 1 above)
Comment (gpacix) [reply deploy/library/postgresql_exec.py:138]: Yes; at some point in the testing process, `ansible-playbook` or `jinja2` was running under Python 3 and complained about the odbc templates.  Unless the collections are very, very long, I doubt we'll even notice the efficiency difference.

File: deploy/roles/odbc/templates/odbcinst.ini (hunk 2 above)
Comment (gpacix) [reply deploy/roles/odbc/templates/odbcinst.ini:33]: Ditto.

**Review by tanderegg — APPROVED** [review]

**Pull request merged**